tools/picard.wdl

```diff
@@ -53,16 +54,17 @@ task mark_duplicates {
 
     runtime {
         memory: memory_gb + " GB"
-        disk: disk_size + " GB"
+        disk: disk_size_gb + " GB"
         docker: 'quay.io/biocontainers/picard:2.27.5--hdfd78af_0'
         maxRetries: max_retries
     }
 }
 
 task validate_bam {
+    # TODO should this be refactored to behave as "default" Picard behaves?
+    #   Default Picard has some weird/not ideal behaviors
+    #   e.g. `max_errors = 100`
```

Comment:
What do you think @adthrasher ? Especially in light of potentially moving to a "community model".

Like the comment says, I don't like Picard's default behavior, and I think we have _some_ room to "improve" the default behavior. But the current state of this task is *very* far from default. And probably too much of a deviation for a "community resource"